packages/ui/primitives/alert.tsx

```diff
@@ -5,21 +5,20 @@ import { VariantProps, cva } from 'class-variance-authority';
 import { cn } from '../lib/utils';
 
 const alertVariants = cva(
-  'relative w-full rounded-lg border p-4 [&>svg]:absolute [&>svg]:text-foreground [&>svg]:left-4 [&>svg]:top-4 [&>svg+div]:translate-y-[-3px] [&:has(svg)]:pl-11',
```

Comment:
had some alignment issues with the icon, copy and pasted the component from `Shadcn/ui`